packages/repository-tests/src/crud/relations/acceptance/belongs-to.relation.acceptance.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect, toJSON} from '@loopback/testlab';
+import {
+  deleteAllModelsInDefaultDataSource,
+  withCrudCtx,
+} from '../../../helpers.repository-tests';
+import {
+  CrudFeatures,
+  CrudRepositoryCtor,
+  CrudTestContext,
+  DataSourceOptions,
+} from '../../../types.repository-tests';
+import {Customer, Order, Shipment} from '../fixtures/models';
+import {givenBoundCrudRepositories} from '../helpers';
+
+export function belongsToRelationAcceptance(
+  dataSourceOptions: DataSourceOptions,
+  repositoryClass: CrudRepositoryCtor,
+  features: CrudFeatures,
+) {
+  describe('BelongsTo relation (acceptance)', () => {
+    before(deleteAllModelsInDefaultDataSource);
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let customerRepo: any;
```

Comment:
This is effectively disabling type checks, it's possible to write e.g `customerRepo.danceMakarena()` and the compiler will happily accept it.

Can we use `EntityCrudRepository<Customer, typeof Customer.prototype.id, {/*relations*/}>` instead?

I think it may be best to create a type alias in `fixtures/models` so that we don't have to repeat the long repository type definition.

```ts
// usage
import {Customer, CustomerRepository} from '../fixtures/models';

// implementation
@model()
export class Customer extends Entity {
  // ...
}

export interface CustomerRepository extends EntityCrudRepository<Customer/*...*/> {
  // define additional members like relation methods here
};
```

Just note that `EntityCrudRepository` is a type, therefore you cannot call `new CustomerRepository`. Feel free to find a better name than `CustomerRepository` to avoid possible confusion about whether the repo is a class or an interface.